modules/op-release-notes-1-14.adoc

```diff
@@ -0,0 +1,269 @@
+=== Operator
+
+* With this update, {pipelines-shortname} supports horizontal pod autoscaling for the operator proxy webhook. If the pod that runs the operator proxy webhook reaches 85% CPU utilization, the autoscaler creates another replica of the pod. If you want to use more than one replica for the operator proxy webhook at startup, you must configure this number in the `options.horizontalPodAutoscalers` spec of the `TektonConfig` CR.
+
+* With this update, the internal leader election for several components of {pipelines-shortname} was improved. The operator controller, operator webhook, proxy webhook, {pac} watcher, {pac} webhook, and the Tekton Chains controller now use separate leader election ConfigMaps. The leader election affects which replica of a component processes a request.
+
+* Before this update, when you scaled up the number of replicas of the {pipelines-shortname} controller, manual intervention was required to enable the use of the new replicas, namely, you needed to delete leases in the leader election. With this update, when you scale up the number of replicas of the {pipelines-shortname} controller, the leader election includes the new replicas automatically, so the new replicas can process information.
+
+* With this update, you can optionally set the following flags in the `spec.pipeline` spec of the `TektonConfig` CR:
+** `coschedule`
+** `enable-cel-in-whenexpression`
+** `enable-param-enum`
+** `enable-step-actions`
+** `enforce-nonfalsifiability`
+** `keep-pod-on-cancel`
+** `max-result-size`
+** `metrics.count.enable-reason`
+** `results-from`
+** `set-security-context`
+** `default-resolver-type`
+
+
+
+[id="triggers-new-features-1-14_{context}"]
+=== Triggers
+
+* With this update, when specifying CEL expressions for the Triggers interceptor, you can use the `first` and `last` functions to access values in a JSON array.
+
+* With this update, when specifying CEL expressions for the Triggers interceptor, you can use the `translate` function that facilitates the utilization of regular expressions to replace characters with specified strings, as in the following example:
++
+.Sample use of the `translate` function
+----
+".translate("[^a-z0-9]+", "ABC")
+----
++
+.Sample input string
+----
+This is $an Invalid5String
+----
++
+.Sample result string
+----
+ABChisABCisABCanABCnvalid5ABCtring
+----
+
+[id="web-console-new-features-1-14_{context}"]
+=== Web console
+
+* With this update, you can enable the web console plugin for {pipelines-shortname}. If you enable the plugin, you can view pipeline and task execution statistics in the *Pipelines overview* page and in the page of a pipeline.
++
+[NOTE]
+====
+To use the web console plugin for {pipelines-shortname}, you must use the following {OCP} point releases or later point releases:
+
+* For {OCP} version 4.12: 4.12.51
+* For {OCP} version 4.13: 4.13.34
+* For {OCP} version 4.14: 4.14.13
+* For {OCP} version 4.15: any point release
```

Comment:
```suggestion
* For {OCP} version 4.15: any release
```
I checked with the peer review team and "point release" isn't a term we normally use - does this suggestion work?